src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -195,6 +209,12 @@ public PersonListPanel getPersonListPanel() {
     private CommandResult executeCommand(String commandText) throws CommandException, ParseException {
         try {
             CommandResult commandResult = logic.execute(commandText);
+
+            if (commandText.startsWith("theme")) {
+                updateTheme(logic.getTheme());
+                helpWindow.updateHelpWindowTheme(logic.getTheme());
+            }
+
```

Comment:
I think this might be a bit "brute force"-ish, but I didn't want to break the abstraction of logic and i needed the scene from MainWindow.fxml